docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -215,38 +236,45 @@ data:
   ...
 ```
 
-Moreover, you have to start the JobManager and TaskManager pods with a service account which has the permissions to create, edit, delete ConfigMaps.
-See [how to configure service accounts for pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for more information.
+此外，必须使用具有创建、编辑、删除 ConfigMap 权限的 service 账号启动 JobManager 和 TaskManager pod。更多信息，请参考[如何为 pod 配置 service 账号](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) 。
+
+当启用了高可用，Flink 会使用自己的 HA 服务进行服务发现。因此，JobManager Pod 会使用 IP 地址而不是 Kubernetes 的 service 名称来作为 `jobmanager.rpc.address` 的配置项启动。完整配置请参考[附录](#appendix)。
+
+<a name="standby-jobManagers"></a>
+
+####  备用 JobManagers
```

Comment:
Keep origianl content?

Reply:
Ok, keep the original content to make it look smoother.